nubi/src/components/header/menu/menu.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+import styles from './styles.module.scss';
+
+function Sidebar() {
+  return (
+    <nav className={styles.menu}>
+      <a href="/" className={styles.menuText}>
```

Comment:
Agrega un comentario para cada link (o uno en general para todos los links del Sidebar), con eso cuando estén listas las URLs para cada elemento no se te olvida. Por ejemplo:
```suggestion
      {/* @TODO:  Reemplazar el href por el apropiado una vez estén listas las secciones referenciadas. */}
      <a href="/" className={styles.menuText}>
```